src/crimson/os/seastore/btree/fixed_kv_btree.h

```diff
@@ -588,6 +588,13 @@ class FixedKVBtree {
                 : true);
             }
           }
+        } else if (child == RESERVATION_PTR) {
+          if constexpr(
+            !std::is_base_of_v<typename internal_node_t::base_t,
+                               child_node_t>) {
+            assert(i->get_val().pladdr.is_paddr());
+            assert(i->get_val().pladdr.get_paddr() == P_ADDR_ZERO);
+          }
```

Comment:
Is `check_node()` supposed to be called any time, including trans pending and committing time?

Reply:
Yes